projects/extension/src/background/index.ts

```diff
@@ -165,6 +176,8 @@ const notifyAllSmoldotCrashErrorChangedListeners = () => {
   })
 }
 
+let extConfigRpcRqId = 0
```

Comment:
The `ConnectionManagerWithHealth` already has this system in place

Reply:
It does but its a private method - so I thought to keep it separated. I can always create a `get` to retrieve the private variable if that sounds ok